### PR TITLE
Add updateRevision param to updateEntry

### PR DESCRIPTION
**Pull request by imsitnikov**

### CODE REVIEW

**Review by stankis — APPROVED** [review]

File: src/services/entry/actions/update-entry.ts [hunk 1]
```diff
@@ -189,6 +192,56 @@ export async function updateEntry(ctx: CTX, updateData: UpdateEntryData) {
...
+                    links: syncedLinks,
+                    createdBy: updatedBy,
+                    updatedBy: updatedBy,
+                })
+                .returning('*')
+                .timeout(DEFAULT_QUERY_TIMEOUT);
+        }
+
+        return revision;
+    };
```
Comment (Sergey-weber) [inline src/services/entry/actions/update-entry.ts:243]: can we separate the method on two different? It's better to maintain, when method has one responsibility.

Comment (imsitnikov) [reply src/services/entry/actions/update-entry.ts:243]: what exactly will these methods do? give an example plz

Comment (Sergey-weber) [reply src/services/entry/actions/update-entry.ts:243]: const revision = isPrivateRoute && updateRevision ? await updateRevision({trx, syncedLinks}) : await createRevision({trx, syncedLinks})

Comment (imsitnikov) [reply src/services/entry/actions/update-entry.ts:243]: [Here](https://github.com/datalens-tech/datalens-us/pull/154/files#diff-94c025171215e845b0908c914d8337abe6dca7f1b144c59004567ef7cc627e23R259) and [here](https://github.com/datalens-tech/datalens-us/pull/154/files#diff-94c025171215e845b0908c914d8337abe6dca7f1b144c59004567ef7cc627e23R354)? Or make three methods?

Comment (Sergey-weber) [reply src/services/entry/actions/update-entry.ts:243]: sure, we can add just in two place - [here](https://github.com/datalens-tech/datalens-us/pull/154/files#diff-94c025171215e845b0908c914d8337abe6dca7f1b144c59004567ef7cc627e23R259) and [here](https://github.com/datalens-tech/datalens-us/pull/154/files#diff-94c025171215e845b0908c914d8337abe6dca7f1b144c59004567ef7cc627e23R354)

**Review by Sergey-weber — APPROVED** [review]

**Review by jhoncool — APPROVED** [review]

**Pull request merged**